src/components/PullRequests/PullRequest.js

```diff
@@ -97,14 +109,20 @@ export const PullRequest = ({ pullRequest }) => {
           </a>
         </div>
 
-        <div className={styles.author}>
-          <img
-            src={pullRequest.author.avatarUrl}
-            alt={pullRequest.author.login}
-            className={styles.authorAvatar}
-          />
-          <p className={styles.authorName}>{pullRequest.author.login}</p>
-        </div>
+        <a
+          href={pullRequest.author.url}
+          target="_blank"
+          rel="noopener noreferrer"
+        >
+          <div className={styles.author}>
+            <img
+              src={pullRequest.author.avatarUrl}
+              alt={pullRequest.author.login}
+              className={styles.authorAvatar}
+            />
+            <p className={styles.authorName}>{pullRequest.author.login}</p>
+          </div>
+        </a>
       </div>
 
       <p className={styles.branches}>
```

Comment:
Would be cool to make this one go to the 'Files Changed' tab of PR, or 2 links to each branch 😉

Reply:
Out of scope 😉 haha. I'll look at adding that too.